### ISSUE TITLE
Add option to delete all screenshots in one go

### ISSUE THREAD

**Issue by senff** [senff/admin-screenshots#2]

**Comment by felipelousantos** [comment]
@senff Do you have any idea how to query the previously taken screenshots?

I came up with some ideas:

* Use the search parameter to query all attachments with "(generated by Admin Screenshots)" on its title — this path doesn't seem super reliable, but it would be quite fast to do it this way.
* Start saving all the screenshot IDs on an option in the database so we can get all these files with these IDs — not super complicated, but it would involve changing the current saving process a bit more.

**Comment by senff** [comment]
All screenshots are saved in the /admin-screenshots folder, so there's that.    It's fairly easy to delete them from the server I'm sure, but then they'd still need to be deleted from the Media Library itself, but it could be a good starting point.

**Comment by felipelousantos** [comment]
Yep, I saw on the code that these were on /admin-screenshots. As you mentioned, the challenge would be to delete them from the Media Library. 

Maybe an option is **not** saving it to the Media Library, which could work even better from a security standpoint and would solve this issue. Is there a specific feature you have in mind for adding to the Media Library? I believe it should work better to just list these on an admin page.
